src/test/groovy/nebula/plugin/publishing/maven/ResolvedMavenPluginIntSpec.groovy

```diff
@@ -58,6 +58,37 @@ class ResolvedMavenPluginIntSpec extends IntegrationSpec {
         asmDep.version.text() == '2.2.3'
     }
 
+    def 'compatible with nebula.provided-base plugin'() {
+        writeHelloWorld('nebula.hello')
+        buildFile << '''
+            buildscript {
+                repositories { jcenter() }
+                dependencies { classpath "com.netflix.nebula:gradle-extra-configurations-plugin:2.2.0" }
```

Comment:
I think we will want to be able to test this without the direct dependency to the plugin. Instead we can just create a new configuration named `provided`.